src/main/java/de/tum/in/www1/artemis/service/dto/athena/TextFeedbackDTO.java

```diff
@@ -0,0 +1,47 @@
+package de.tum.in.www1.artemis.service.dto.athena;
+
+import javax.validation.constraints.NotNull;
+
+import de.tum.in.www1.artemis.domain.Feedback;
+import de.tum.in.www1.artemis.domain.TextBlock;
+import de.tum.in.www1.artemis.domain.TextBlockRef;
+import de.tum.in.www1.artemis.domain.TextSubmission;
+
+/**
+ * A DTO representing a Feedback, for transferring data to Athena
+ */
+public record TextFeedbackDTO(long id, long exerciseId, long submissionId, String text, String detailText, double credits, Integer indexStart, Integer indexEnd) {
```

Comment:
This could lead to issues when tutors use structured grading instructions. Here the feedback is not only the detail text, but also created by the grading instruction. If Athena decides to apply such a feedback to another submission, the grading instruction context gets lost.

Reply:
I've added a new `grading_instruction` field for feedback in Athena as that seemed like the most straightforward solution to the problem.